src/bcf-toolkit/Converter/Bcf30/SchemaConverterToBcf21.cs

```diff
@@ -1,11 +1,296 @@
-using System;
-using BcfToolkit.Model;
+using System.Linq;
+using BcfToolkit.Builder.Bcf21;
+using BcfToolkit.Model.Bcf21;
+using BcfBuilder = BcfToolkit.Builder.Bcf21.BcfBuilder;
+using BimSnippetBuilder = BcfToolkit.Builder.Bcf21.BimSnippetBuilder;
+using BitmapBuilder = BcfToolkit.Builder.Bcf21.BitmapBuilder;
+using CommentBuilder = BcfToolkit.Builder.Bcf21.CommentBuilder;
+using ComponentBuilder = BcfToolkit.Builder.Bcf21.ComponentBuilder;
+using DocumentReferenceBuilder = BcfToolkit.Builder.Bcf21.DocumentReferenceBuilder;
+using LineBuilder = BcfToolkit.Builder.Bcf21.LineBuilder;
+using ClippingPlaneBuilder = BcfToolkit.Builder.Bcf21.ClippingPlaneBuilder;
+using MarkupBuilder = BcfToolkit.Builder.Bcf21.MarkupBuilder;
+using VisibilityBuilder = BcfToolkit.Builder.Bcf21.VisibilityBuilder;
+using VisualizationInfoBuilder = BcfToolkit.Builder.Bcf21.VisualizationInfoBuilder;
 
 namespace BcfToolkit.Converter.Bcf30;
 
 public static class SchemaConverterToBcf21 {
 
-  public static Model.Bcf21.Bcf Convert(IBcf from) {
-    throw new NotImplementedException();
+  /// <summary>
+  ///   This method translates the specified object with a BCF version to the
+  ///   desired one.
+  ///   From:   [BCF 3.0]
+  ///   To:     [BCF 2.1]
+  /// </summary>
+  /// <param name="from">The object which must be converted.</param>
+  /// <returns>Returns the converted object.</returns>
+  public static Model.Bcf21.Bcf Convert(Model.Bcf30.Bcf from) {
+    var builder = new BcfBuilder();
+    builder
+      .AddMarkups(from.Markups.Select(ConvertMarkup).ToList())
+      .SetProject(ConvertProject(from.Project?.Project));
+
+    return builder.Build();
```

Comment:
Where the builder functions are chained, the final `Build` can be called in the pipe.
```
return builder
  .AddMarkups(from.Markups.Select(ConvertMarkup).ToList())
  .SetProject(ConvertProject(from.Project?.Project));
  .Build();
```

Reply:
Ok. Fixed them.